src/schema/v2/lot.ts

```diff
@@ -0,0 +1,179 @@
+import { GraphQLError, GraphQLObjectType } from "graphql"
+import { SaleArtworkType } from "schema/v2/sale_artwork"
+import { Response as FetchResponse } from "node-fetch"
+import { ResolverContext } from "types/graphql"
+import {
+  NodeInterface,
+  SlugAndInternalIDFields,
+} from "schema/v2/object_identification"
+import { pageable } from "relay-cursor-paging"
+import { connectionWithCursorInfo } from "./fields/pagination"
+import gql from "lib/gql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { connectionFromArray } from "graphql-relay"
+
+export const LotType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Lot",
+  description:
+    "A lot in an auction containing merged Sale artwork and Lot state data",
+  interfaces: () => {
+    return [NodeInterface]
+  },
+  fields: () => {
+    return {
+      // Must place an `id` at the root of the object (next to saleArtwork)
+      // when resolving a lot for this to work (see resolver)
+      ...SlugAndInternalIDFields,
+      saleArtwork: {
+        type: SaleArtworkType,
+        description: "The saleArtwork object.",
+        resolve: ({ saleArtwork }) => {
+          return saleArtwork
+        },
+      },
+    }
+  },
+})
+
+/**
+ * Fetch relevant sale artworks and use them to fetch lot states
+ * from causality. Then pass the lot states through to the stitching layer
+ * via the graphql request context.
+ */
+const resolveWatchedLotConnection = async (_parent, args, context) => {
```

Comment:
Lot resolution and stitching code is all going in this file, with the relevant pieces brought into either the local schema or the stitched causality extension schema later. This keeps highly coupled code in the same file and reduces noise in confusing stitching files. This is the resolver...

Reply:
#minor i would name this `watchedLotConnectionResolver` to stick more with the pattern (resolver at the end).